trajectory_msgs/MultiDOFJointTrajectory desired_position_q
trajectory_msgs/MultiDOFJointTrajectory desired_position_g
trajectory_msgs/MultiDOFJointTrajectory desired_position_s
CommandArmServoAngleTrajectory          desired_arm_rad

